src/helpers/digital-exchange/components.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import { formattedText } from '@entando/utils';
+import { Button } from 'patternfly-react';
+
+// eslint-disable-next-line import/prefer-default-export
```

Comment:
this is also only necessary if there is just one named export within a file. please remove the rule disabling if it is not necessary